kbase-extension/static/kbase/js/widgets/narrative_core/catalog/catalog_util.js

```diff
@@ -0,0 +1,108 @@
+define([
+    'bluebird',
+    'jquery'
+], function (Promise, $) {
+    'use strict';
+
+    function CatalogUtil() {
+
+        // edited from: http://stackoverflow.com/questions/3177836/how-to-format-time-since-xxx-e-g-4-minutes-ago-similar-to-stack-exchange-site
+        this.getTimeStampStr = function (objInfoTimeStamp) {
```

Comment:
Good call!  I didn't have these util methods (or at least I didn't know where they are) in kbase-ui, but I'll update this in the narrative.  I can probably get rid of this util code altogether.